common/djangoapps/request_cache/middleware.py

```diff
@@ -1,3 +1,9 @@
+"""
+An implementation of a RequestCache. This cache is reset at the beginning
+and end of every request.
+"""
+
+import crum
```

Comment:
Do we gain much by switching over to `crum` for this functionality? It seems like it's been abandoned for a couple of years.